backup/moodle2/backup_carousel_block_task.class.php

```diff
@@ -0,0 +1,53 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Carousel block backup task class
+ *
+ * @package   block_carousel
+ * @copyright 2021 Nicholas Hoobin (nicholashoobin@catalyst-au.net)
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+require_once($CFG->dirroot . '/blocks/carousel/backup/moodle2/backup_carousel_stepslib.php');
```

Comment:
might be a little cleaner as __DIR__ . '/backup_carousel_stepslib.php. Personal preference wins here so either or